integreat_cms/cms/templates/_form_language_tabs.html

```diff
@@ -76,14 +76,14 @@
             <li class="z-10 ml-auto">
                 <div id="language-switcher" class="flex flex-col text-black">
                     <div id="language-switcher-current"
-                         class="font-bold text-right bg-water-500 py-[calc(0.75rem+2px)] px-4 border-l border-t border-r border-blue-500 rounded-t cursor-default">
+                         class="flex font-bold text-right bg-water-500 py-[calc(0.75rem+2px)] px-4 border-l border-t border-r border-blue-500 rounded-t cursor-default">
                         <i id="language-switcher-globe" icon-name="globe" class="hidden"></i>
                         <span id="language-switcher-text">{% translate "Other Languages" %}</span>
                         <i icon-name="chevron-down"></i>
                     </div>
-                    <ul id="language-switcher-list" class="z-10 hidden h-0">
-                    </ul>
                 </div>
+                <ul id="language-switcher-list" class="z-10 hidden h-0">
+                </ul>
```

Comment:
```suggestion
                <ul id="language-switcher-list" class="z-10 hidden h-0" />
```

Reply:
Unfortunately this doesn't work. It seems to break the entire view (s. screenshot)
![grafik](https://github.com/user-attachments/assets/900286e8-5d68-47af-9ec8-0936c2cb16b0)
I hope it's okay if I keep it the way it is :)